x-pack/plugins/fleet/server/routes/agent/handlers.ts

```diff
@@ -33,6 +43,8 @@ import type {
 import { defaultIngestErrorHandler } from '../../errors';
 import * as AgentService from '../../services/agents';
 
+import { setCachedFile, getCachedVersionFile } from './utils';
```

Comment:
Looks like the module is name `version.utils.ts` it probably should be renamed `version_utils.ts` (to follow file naming scheme) and you should import from `./version_util` here